irohad/model/impl/model_operators.cpp

```diff
@@ -187,7 +187,8 @@ namespace iroha {
       return std::equal(commands.begin(), commands.end(), rhs.commands.begin(),
```

Comment:
I guess it is not necessary. Just calculate hashes of 2 transactions and compare them.